setzer/dialogs/bibtex_wizard/pages/page_create_new_entry.py

```diff
@@ -87,14 +88,14 @@ def __init__(self, document_types):
             label.set_line_wrap(True)
             markup = '<b>@' + document_type + '</b>\n\n'
             markup += attributes['description'] + '\n\n'
-            markup += 'Required fields: '
+            markup += _('Required fields') + ': '
```

Comment:
Same here. `Required fields` appears at least twice, so let's separate the colon.